model/item.cc

```diff
@@ -144,7 +144,8 @@ namespace minsky
      
     // then, check whether a resize handle has been selected  
     float z=zoomFactor();
-    if ((abs(x-left()) < portRadius*z || abs(x-right()) < portRadius*z) &&
+    if (!variableCast() && !operationCast() &&
```

Comment:
Could we not provide an overridden clickType in VariableBase and OperationBase?

Reply:
Sorry, I missed this comment. I saw that the differences between clickType in Operationbase and the Item class were minimal, which is why I delegated to the method in the base class in that case.

VariableBase has its own clickType but still calls Item::clickType() at the end in "  return Item::clickType(xx,yy);". So, if a case is not fully covered, e.g., for the coupled it op when the resize handles are activated as one moves the mouse over the top of the bounding box midway between the int op and attached var, then Item::clickType() gets called. So, to avoid that, I issue a !variableCast() here. The alternative is to remove the final "  return Item::clickType(xx,yy);" in the overridden members in VariableBase, OperationBase, PlotWidget, etc. I don't know which is better but assumed that, since clickType is expensive, using as few of them as possible would be the way to go?